tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Services/IdentityServiceTests.cs

```diff
@@ -3,6 +3,7 @@
 using FluentAssertions;
 using NHSD.BuyingCatalogue.Ordering.Api.Services;
 using NHSD.BuyingCatalogue.Ordering.Api.UnitTests.AutoFixture;
+using NHSD.BuyingCatalogue.Ordering.Contracts;
```

Comment:
Is this `using` statement required?

Reply:
Removed `using` statement `NHSD.BuyingCatalogue.Ordering.Contracts`